bitsandbytes/backends/cuda.py

```diff
@@ -0,0 +1,479 @@
+import ctypes as ct
```

Comment:
@Titus-von-Koeller Can you run `ruff format` "pre-emptively" on this file? Since it's a big new file, we could have it adhere to the future format at this point and avoid an additional commit reformatting it :)

Reply:
Yes, I'll do that. We should also set up ruff formatting for the rest of the repo within the coming two weeks. That would be a big step forward and make collab easier. We have such active contributions, this would really homogenize things.